src/Exceptions/HttpServiceUnavailable.php

```diff
@@ -8,4 +8,3 @@ class HttpServiceUnavailable extends HttpException {
     public const DESCRIPTION = "Indicates that the server is currently unable to handle the request due to a temporary overload or scheduled maintenance, which will likely be alleviated after some delay.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces